### ISSUE TITLE
Fortran interface in separate repository 

### ISSUE THREAD

**Issue by Nicholaswogan** [LLNL/sundials#301]
It would be useful if the Fortran interface was in a separate repository or if it could be built without building the C part of sundials.

I have a package that I'm putting on conda-forge (https://github.com/conda-forge/staged-recipes/pull/23182#issuecomment-1629533181). It depends on sundials, but it requires the fortran interface. I can not link against the conda-forge sundials because it does not build the fortran interface. I need to instead, independent of conda-forge, build sundials with the fortran interface and link to my package.

Anyway, it would be useful to separate Sundials and the Fortran interface in some way.

**Comment by balos1** [comment]
So what you want is to be able to build the Fortran interface by linking to an existing installation of the C libraries?

**Comment by Nicholaswogan** [comment]
Yes. I want to build the fortran interface, then link to an installation of sundials.

**Comment by Nicholaswogan** [comment]
This could also be achieved with adjustments to the CMakelists.txt files

**Comment by balos1** [comment]
I think adjustments to the CMake would be the way to go. I have marked this as a feature-request.

**Comment by balos1** [comment]
The CMake system is indeed a bit complicated. You can attempt it if you would like, and a contribution in this regard would be appreciated. I cannot promise that someone from the sundials team will be able to implement this in the near future.
